src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helpers/StringUtils.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.helpers
+
+import com.google.common.base.Utf8
+
```

Comment:
rewrote this so it only truncates enough so it fits, else Strings with lots of quad-byte characters at beginning of String were being truncate too much.,
Also deal with truncating half-way through a multi-byte character by accident